Tchap/Modules/Room/Members/RoomParticipantsViewController.m

```diff
@@ -902,36 +902,39 @@ - (void)checkExpiredAccounts:(void (^)(void))completion
     dispatch_group_t requestsGroup = dispatch_group_create();
```

Comment:
You should remove this dispatch_group_t, it is useless now

Reply:
removed